superflore/generators/ebuild/ebuild.py

```diff
@@ -142,9 +142,9 @@ def get_ebuild_text(self, distributor, license_text, die_msg=None):
                     ret += get_license(self.upstream_license) + "\"\n\n"
             elif isinstance(self.upstream_license, list):
                 ret += "LICENSE=\"( "
-                for l in self.upstream_license:
-                    l = get_license(l)
-                    ret += '{0} '.format(l)
+                for _ul in self.upstream_license:
```

Comment:
This doesn't need a for loop, use list comprehension.

```
ret += ' '.join([get_license(ul) for ul in self.upstream_license])
```

Reply:
That's a good tip -- thanks!